docs/features/security/auth/shared-access-key.md

```diff
@@ -8,7 +8,7 @@ layout: default
 The `Arcus.WebApi.Security` package provides a mechanism that uses shared access keys to grant access to a web application.
 This authentication process consists of two parts:
 
-1. Find the configured HTTP request header that contains the shared access key
+1. Find the configured HTTP request header or the configured Query parameter that contains the shared access key
```

Comment:
I think we should explain more how this works with examples. "Either header or query" accompanied with filter/attribute pair for each. Now the example shows only the combination where everything is filled out.

We should also mention that when query and/or header is filled out, and we have configured header and/or query, how the application will respond.

Reply:
I'm skipping docs for now until feature is done and will then do a small sweep

Reply:
I would indeed be a bit more clear here on the scenarios where you can:
1. Use header only
2. Use query parameter only
3. Support both
4. It will block if both are specified but only one is correct, although both are specified.

Reply:
Yes, agreed. My initial comment is also still valid for this. It's not so bad that we don't have a complete docs-set for this, but initial seperation header/query would be the minimum for me.